sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -3602,6 +3914,52 @@ public partial class PrometheusForwarderDataSource : System.ClientModel.Primitiv
+        Azure.ResourceManager.Monitor.Models.Receiver System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Receiver>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Receiver>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Receiver>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ReceiverType : System.IEquatable<Azure.ResourceManager.Monitor.Models.ReceiverType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ReceiverType(string value) { throw null; }
+        public static Azure.ResourceManager.Monitor.Models.ReceiverType Ama { get { throw null; } }
+        public static Azure.ResourceManager.Monitor.Models.ReceiverType Otlp { get { throw null; } }
+        public static Azure.ResourceManager.Monitor.Models.ReceiverType PipelineGroup { get { throw null; } }
+        public static Azure.ResourceManager.Monitor.Models.ReceiverType Syslog { get { throw null; } }
+        public static Azure.ResourceManager.Monitor.Models.ReceiverType UDP { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Monitor.Models.ReceiverType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Monitor.Models.ReceiverType left, Azure.ResourceManager.Monitor.Models.ReceiverType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Monitor.Models.ReceiverType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Monitor.Models.ReceiverType left, Azure.ResourceManager.Monitor.Models.ReceiverType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class RecordMap : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.RecordMap>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.RecordMap>
```

Comment:
this name is quite generic, could we add some prefix to specify it?

Reply:
Prefixed `SchemaMap/RecordMap/ResourceMap/ScopeMap` with `MonitorWorkspaceLogs`.